Modules/_blake2/blake2b_impl.c

```diff
@@ -394,44 +394,18 @@ py_blake2b_dealloc(PyObject *self)
     PyObject_Del(self);
 }
 
+static PyType_Slot blake2b_type_slots[] = {
+    {Py_tp_dealloc, py_blake2b_dealloc},
```

Comment:
When you convert a static type to a heap type, you have to modify its tp_dealloc slot to DECREF its type.